### PR TITLE
Improve RDM support some more.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/src/transceiver.c [hunk 1]
```diff
@@ -82,14 +87,15 @@ TransceiverData g_transceiver;
 static inline uint16_t Transceiver_MicroSecondsToTicks(uint16_t micro_seconds) {
   return micro_seconds * (SYS_CLK_FREQ / 1000000);
 }
+
 /*
```
Comment (peternewman) [inline firmware/src/transceiver.c]: **

File: firmware/src/transceiver.c [hunk 2]
```diff
@@ -82,14 +87,15 @@ TransceiverData g_transceiver;
...
  * @brief Convert 10ths of a millisecond to ticks.
+ * @param time A value between 0 and 650 (6.5mS).
  */
 static inline uint16_t Transceiver_TenthOfMilliSecondsToTicks(uint16_t time) {
   return 100 * time * (SYS_CLK_FREQ / 1000000 / 8);
 }
 
-
 /*
```
Comment (peternewman) [inline firmware/src/transceiver.c]: **

File: firmware/src/transceiver.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/transceiver.c]: Again ,these are private.

**Comment by peternewman** [comment]
Retrospective LGTM

**Pull request merged**